docker-compose.yml

```diff
@@ -47,13 +69,20 @@ services:
       DB_DATABASE: postgres
       DB_HOST: postgres
       DB_PORT: 5432
+      START_BLOCK: 1
+      # anything that is not "true" will avoid start nodemon
+      WATCH: true
+      # this will manage which dotenv file will be used on every reload
+      NODE_ENV: develop
     volumes:
       - ./:/app
+      - ignore_dist:/app/dist
+      - ignore_types:/app/src/types
     command:
       - ${SUB_COMMAND:-} # set SUB_COMMAND env variable to "test" to run tests
       - -f=/app
       - --db-schema=app
-      - --workers=4
+      - --workers=1
```

Comment:
Why switch to 1 worker?

Reply:
For development (with hot-reload, which is by default true) spam 4 workers produce tons more logs which complicates the debugging.

Reply:
#PUC, just add that as a comment here please. :+1: